test/lib/identity_code_test.rb

```diff
@@ -30,4 +30,15 @@ def test_is_valid_randomly_for_estonian_id_codes
     identity_code = IdentityCode.new('EE', array_of_codes.sample)
     assert(identity_code.valid?)
   end
+
+  def test_is_valid_for_every_century
```

Comment:
- Looking at the name of the test, it's not clear what this test actually tests.
- I also find the tests themselves quite cryptic: why value `14501234213` is valid?
- Don't we need to test negative case (when a record is invalid)?

Reply:
- Agreed. This test can be named more accurately.
- For me it's self-explanatory, I assert that this ID code is valid. Why would one write a test that intentionally is not valid? 
- Yup, great idea.